### ISSUE TITLE
Burp Error While Adding Extension

### ISSUE THREAD

**Issue by arbazkiraak** [silentsignal/burp-image-size#1]
java.lang.ClassNotFoundException: burp.BurpExtender
    at java.net.URLClassLoader$1.run(Unknown Source)
    at java.net.URLClassLoader$1.run(Unknown Source)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.net.URLClassLoader.findClass(Unknown Source)
    at java.lang.ClassLoader.loadClass(Unknown Source)
    at java.lang.ClassLoader.loadClass(Unknown Source)
    at java.lang.Class.forName0(Native Method)
    at java.lang.Class.forName(Unknown Source)
    at burp.u1g.a(Unknown Source)
    at burp.u1g.<init>(Unknown Source)
    at burp.ksb.a(Unknown Source)
    at burp.bnb.run(Unknown Source)
    at java.lang.Thread.run(Unknown Source)

**Comment by dnet** [comment]
What JAR file are you using? Is it one of the [pre-built releases](https://github.com/silentsignal/burp-image-size/releases) or did you compile it on your own? If so, how did you do it? Can you attach your JAR file to this issue? What operating system and what version of Burp and Java (JDK, JRE) are you using?

**Comment by gamermount56** [comment]
I am getting same error.

I am using window 8.1

Burp Version : Burp suite Pro 1.6v

**Comment by dnet** [comment]
@gamermount56 thanks, but please also answer the other, more important questions:

> What JAR file are you using? Is it one of the [pre-built releases](https://github.com/silentsignal/burp-image-size/releases) or did you compile it on your own? If so, how did you do it? Can you attach your JAR file to this issue?

**Comment by indianfalcon** [comment]
Had the same issue - updated java to version 8 and it worked - you can follow instructions here:
http://www.tecmint.com/install-java-jdk-jre-in-linux/

**Comment by dnet** [comment]
Thanks, I compiled the pre-built releases with JDK version 8 (producting bytecode version 52 files), which indeed cannot be loaded into a version 6 or 7 JVM -- even though the source itself can be compiled even with JDK version 6, which is also [validated by Travis CI](https://github.com/silentsignal/burp-image-size/blob/master/.travis.yml).

I compiled v0.3 for version 6 as well (bytecode version 50), and added it to the [v0.3 release](https://github.com/silentsignal/burp-image-size/releases/tag/v0.3), select the `burp-image-size-v0.3-java1.6.jar` and it should work for version 6 and 7 JVMs as well. Thanks @indianfalcon for pointing this out!
